mne/io/base.py

```diff
@@ -33,17 +33,173 @@
+                    for i in e]
+            else:
+                # volume source estimates
+                col_names = ['VOL {0}'.format(vert) for vert in self.vertices]
+        else:
+            if isinstance(self, _BaseEpochs):
+                picks = self._get_check_picks(picks, self.ch_names)
+                default_index = ['condition', 'epoch', 'time']
+                data = self.get_data()[:, picks, :]
+                times = self.times
+                n_epochs, n_picks, n_times = data.shape
+                data = np.hstack(data).T  # (time*epochs) x signals
+
+                # Multi-index creation
+                mindex.append(('time', np.tile(times, n_epochs)))
+                id_swapped = dict((v, k) for k, v in self.event_id.items())
+                names = [id_swapped[k] for k in self.events[:, 2]]
+                mindex.append(('condition', np.repeat(names, n_times)))
+                mindex.append(('epoch',
+                              np.repeat(np.arange(n_epochs), n_times)))
+                col_names = [self.ch_names[k] for k in picks]
+
+            elif isinstance(self, (RawFIFF, RawArray, Evoked)):
+                picks = self._get_check_picks(picks, self.ch_names)
+                default_index = ['time']
+                if isinstance(self, (RawFIFF, RawArray)):
+                    data, times = self[picks, start:stop]
+                elif isinstance(self, (Evoked)):
+                    data = self.data[picks, :]
+                    times = self.times
+                    n_picks, n_times = data.shape
+                data = data.T
+                mindex.append(('time', times))
+                col_names = [self.ch_names[k] for k in picks]
+
+            types = [channel_type(self.info, idx) for idx in picks]
+            n_channel_types = 0
+            ch_types_used = []
+
+            scalings = _mutable_defaults(('scalings', scalings))[0]
+            for t in scalings.keys():
+                if t in types:
+                    n_channel_types += 1
+                    ch_types_used.append(t)
+
+            for t in ch_types_used:
+                scaling = scalings[t]
+                idx = [picks[i] for i in range(len(picks)) if types[i] == t]
+                if len(idx) > 0:
+                    data[:, idx] *= scaling
+
+        if index is not None:
+            _check_pandas_index_arguments(index, default_index)
+        else:
+            index = default_index
+
+        if copy is True:
+            data = data.copy()
+
+        times = np.round(times * scale_time)
+
+        assert all(len(mdx) == len(mindex[0]) for mdx in mindex)
+
+        df = pd.DataFrame(data, columns=col_names)
+        [df.insert(i, k, v) for i, (k, v) in enumerate(mindex)]
+        if index is not None:
+            if 'time' in index:
+                print('Converting time column to int64...')
+                df['time'] = df['time'].astype(np.int64)
```

Comment:
It use to be logged and @Eric89GXL had some concerns about this. I am happy to change back...